extensions/visualizations/oppia-visualization-frequency-table.directive.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
This file is diffed as add-delete rather than modify-rename. Please have it in the latter format.